apps-rendering/src/components/interactiveAtom.tsx

```diff
@@ -30,13 +31,13 @@ const InteractiveAtomStyles = (
 	}
 `;
 const atomCss = `
-    ${pageFonts} 
-    
+    ${pageFonts}
+
     @media (prefers-color-scheme: dark) {
         body {
             background: white;
             padding: ${remSpace[3]} !important;
-        } 
+        }
```

Comment:
This is obviously not types -> libs related. I think it must have happened when I ran `npm run lint -- --fix`